src/shared/utils/workspace.ts

```diff
@@ -53,11 +53,11 @@ export function tryProcessBazelFlutterConfig(logger: Logger, config: WritableWor
 			return path.join(bazelWorkspaceRoot, relOrAbsolute);
 		}
 
-		function makeScript(relOrAbsolute: string | undefined, replacesArgs = 1): CustomScript | undefined {
+		function makeScript(relOrAbsolute: string | undefined, replacesArgs = 1, makeFull = true): CustomScript | undefined {
```

Comment:
I can't see any code passing `false` for `makeFull` here - should there be?

Reply:
Old code, thanks